examples/step-63/step-63.cc

```diff
@@ -297,7 +337,15 @@ namespace Step63
   }
 
 
+  // @sect3{Right-hand Side and Boundary Values}
 
+  // The problem solved in this tutorial is an adaptation of Ex. 3.1.3
+  // found on pg. 118 of Finite Elements and Fast Iterative Solvers:
+  // with Applications in Incompressible Fluid Dynamics by Elman, Silvester,
+  // and Wathen. The main difference being that we add a hole in the center
+  // of our domain with zero Dirichlet boundary.
+
```

Comment:
@kronbichler @tjhei  Is this acceptable for a citation? Should I include a link instead of title/authors?

Reply:
I think it is good to give the book citation. You might add a link underneath it, e.g. by `<a href="https://global.oup.com/academic/product/finite-elements-and-fast-iterative-solvers-9780199678808">...with Applications in Incompressible Fluid Dynamics by Elman, Silvester, and Wathen</a>`